glossary.yml

```diff
@@ -5726,6 +5726,10 @@
     term: ግጥም ጥርብ ግራፍ
     def: >
       የቁጥር ውሂብ ስብስብ ስርጭት ግራፊክ ውክልና ፣ ብዙውን ጊዜ ቀጥ ያለ የአሞሌ ግራፍ
+ uk:
```

Comment:
Thank you for your first contribution @mariyaspatar! YAML is very sensitive to indentation, that's why you have a notification from GitHub actions about the syntax error. You can fix that, but editing a copy of this file in your fork in the branch corresponding to this pull request - you can find it at https://github.com/mariyaspatar/glosario/blob/mariyaspatar-glosario-1/glossary.yml. Edit it, commit changes, and the pull request will be automatically updated with the new version of the file. You can write something like "Fix indentation" etc. in the commit message.

Reply:
@mariyaspatar thanks - the fix passes the tests.